sdk/communication/tests.yml

```diff
@@ -1,5 +1,3 @@
-trigger: none
```

Comment:
Please re-add `trigger: none`. While the definition should be disabled by default, I would be much more comfortable if there were no accidents.